### PR TITLE
Cleanup for the media tracking PR

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
- update the dev app for testing media APIs
- small cleanups for the previous media tracking PR

### CODE REVIEW

File: code/AEPSDK.brs [hunk 1]
```diff
@@ -297,8 +297,11 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
                 _adb_logError("createMediaSession() - Cannot create media session, invalid XDM data")
                 return
             end if
-
-            sessionId = m._private.mediaSession.startNewSession()
+            playhead = 0
```
Comment (addb) [inline code/AEPSDK.brs:300]: There is a position variable on L285, can we check if the playhead is valid there and combine this logic.

Comment (yangyansong-adbe) [reply code/AEPSDK.brs:300]: The L285 code is used to handle the position/playhead of the previous session. It might be confusing to mix them together.

Comment (addb) [reply code/AEPSDK.brs:300]: We will get getting Playhead from customer right? So ideally we dont need any playhead logic. This is for fallback? What about sessionEnd?

**Review by addb — APPROVED** [review]

File: code/AEPSDK.brs (hunk 1 above)
Comment (yangyansong-adbe) [reply code/AEPSDK.brs:300]: Discussed offline and decided to move this logic from the API layer to the SDK (task node) layer in another PR.

**Pull request merged**